scanner/src/main/kotlin/scanners/scancode/ScanCodeResultParser.kt

```diff
@@ -42,6 +44,8 @@ import org.ossreviewtoolkit.utils.spdx.toSpdxId
 
 import org.semver4j.Semver
 
+const val MAX_SUPPORTED_OUTPUT_FORMAT_VERSION = "2.0.0"
```

Comment:
Should this remain internal?

Reply:
I was considering that, but then though programmatic users of the scanner library might want to access this.